app/views/unloading_permission/components/table_2/item/goods.scala.xml

```diff
@@ -18,7 +18,7 @@
             <fo:table-cell font-weight="bold" padding-left="3mm" wrap-option="wrap">
                 <!-- Container No(s) -->
                 <fo:block>
-                    @permission.goodsItems.head.containers.head
+                    @permission.goodsItems.head.containers.headOption
```

Comment:
Same here

Reply:
We know we're only here it a single item exits AND a single container (or no container). So headOption either displays a value or not